layouts/shortcodes/tooltip.html

```diff
@@ -1,6 +1,6 @@
 <!-- Create array/map of all possible tooltips. -->
 {{ $tooltips := dict
-    "DataCap" "An asset that can be attached to a storage deal to increase the deal's value for storage providers."
+    "DataCap" "An resource that can be used to make storage deals which increases the deal's value for storage providers by a factor of 10."
```

Comment:
```suggestion
    "DataCap" "A resource that can be added storage deals, increasing that deal's value for storage providers by a factor of 10."
```

I've not dealt directly with Fil+ for a while, but from my understanding it's something that is _added_ to a deal to make it more valuable to an SP. If I have DataCap available, I can choose to not apply it to certain deals.

Is this correct @dkkapur?

I also tweaked your wording a teeny bit.